docs/pages/try-out-teleport/docker-compose.mdx

```diff
@@ -52,8 +52,8 @@ Let's use `docker-compose` to start Teleport demo lab - a configured local clust
 ```code
 # Download the quick start file from our GitHub repo
 $ curl -Lso teleport-lab.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-lab.yml
-# Start teleport demo lab using docker-compose
-$ docker-compose -f teleport-lab.yml up -d
+# Pull the latest image and start the teleport demo lab using docker-compose
+$ docker-compose -f teleport-lab.yml pull && docker-compose -f teleport-lab.yml up -d
```

Comment:
I haven't tested this, but it looks like there's a `--pull always` flag on `docker-compose up` (https://docs.docker.com/engine/reference/commandline/compose_up/)—does that work the same way?

Reply:
Interestingly, I tested this and at least for me, with Docker for Mac, it doesn't actually always force pull 🤕 .  I tested by renaming a faulty image as `teleport-lab:11` and run the compose and it failed (since compose logs don't give much).  After forcing a local `docker pull...` to overwrite my bad image, it started up.